### PR TITLE
Better way to filter todos checked

### PR DESCRIPTION
**Pull request by DCKT**
Hi,

I found a little messy to use twice the same operation instead of using a variable for options.

What do you think ?

### CODE REVIEW

File: examples/simple-todos/simple-todos.js [hunk 1]
```diff
@@ -19,13 +19,13 @@ if (Meteor.isClient) {
...
     tasks: function () {
+      var options = {};
+
       if (Session.get("hideCompleted")) {
-        // If hide completed is checked, filter tasks
-        return Tasks.find({checked: {$ne: true}}, {sort: {createdAt: -1}});
-      } else {
-        // Otherwise, return all of the tasks
-        return Tasks.find({}, {sort: {createdAt: -1}});
+        options = {checked: {$ne: true}};
```
Comment (glasser) [inline examples/simple-todos/simple-todos.js]: I'm not sure if we should take this, but if we do we should not call this variable options: options is the name of the second argument to `find`, not the first.

**Comment by rclai** [comment]
Call it `criteria` or `filter`.

**Comment by DCKT** [comment]
@rclai @glasser You are right, filter looks like a greater name for this task

**Comment by avital** [comment]
I personally don't think this is an improvement. At the very least the original code had comments. And it was simpler to see "if X, return this query; otherwise return another one". Curious what @stubailo thinks.

**Comment by stubailo** [comment]
I think this should be an issue here: https://github.com/meteor/tutorials

I don't think it's really a big enough improvement - anyone who knows about JavaScript will be able to make this abstraction themselves I think.

**Pull request closed without merging**